crates/core/src/document/pdf.rs

```diff
@@ -166,7 +166,16 @@ impl Document for PdfDocument {
     }
 
     fn pages_count(&self) -> usize {
-        unsafe { mp_count_pages(self.ctx.0, self.doc) as usize }
+        unsafe {
+            let page_count: i32 = mp_count_pages(self.ctx.0, self.doc);
+
+            if page_count <= 0 {
+                eprintln!("Error: MuPDF Fitz count_pages function returned {}.", page_count);
```

Comment:
[_MuPDF_ prints errors on _stderr_](https://github.com/ArtifexSoftware/mupdf/blob/336ee9e30cfc2bd59bf75320ce2c4455aa1efd42/source/fitz/error.c#L47), so this shouldn't be necessary.